Ginger/Ginger/RunSetLib/RunSetOperations.cs

```diff
@@ -37,21 +38,31 @@ public static RunSetConfig CreateNewRunset(string runSetName="", RepositoryFolde
                 if (!InputBoxWindow.GetInputWithValidation(string.Format("Add New {0}", GingerDicser.GetTermResValue(eTermResKey.RunSet)), string.Format("{0} Name:", GingerDicser.GetTermResValue(eTermResKey.RunSet)), ref runSetName, System.IO.Path.GetInvalidPathChars()))
                 {
                     return null;
-                }                    
+                }
+
+                while (WorkSpace.Instance.SolutionRepository.GetAllRepositoryItems<RunSetConfig>().Where(r => r.ItemName.ToLower() == runSetName.ToLower()).FirstOrDefault() != null)
+                {
+                    Reporter.ToUser(eUserMsgKeys.DuplicateRunsetName, string.Format("'{0}' already exists, please use different name", runSetName));
```

Comment:
the actual message should be set in "UserMessagesPool" only the name of the Runset should be sent as parameter